tutorials/system-building-protein-ligand.ipynb

```diff
@@ -94,7 +94,6 @@
     "prot = Molecule('3PTB')\n",
     "# Or use the pdb file found in the HTMD data directory\n",
     "prot = Molecule(join(datadir, 'trypsin.pdb'))\n",
-    "prot.renumberResidues()"
```

Comment:
I shouldn't have accepted this, it breaks nb format, as the previous line ends in comma. Going to push now a fix